scipy/interpolate/_ndgriddata.py

```diff
@@ -110,6 +115,8 @@ def __call__(self, *args):
             or x1 can be array-like of float with shape ``(..., ndim)``
 
         """
+        # This function doesn't use NDInterpolatorBase._set_xi since it performs
+        # unrequired calculations.
```

Comment:
This seems to be a leftover.

Reply:
Actually, it isn't. I wanted to explain why this subclass has its own implementation for this function.

Reply:
Took me a while to figure out what is referred to as this function. But this is minor, let's sort out the big picture first (cf my other comment about the `__call__).

Reply:
Please consider rephrasing to clarify what is "this function".